### DISCUSSION TITLE
XnView MP

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#28] [General]
# XnView MP

Check if XnView MP is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/118502eb-dd16-413b-9a49-54b21329bf4e

**Comment by talynone** [comment]
The zip version works according to the app dev:

https://newsgroup.xnview.com/viewtopic.php?t=47469

**Reply by OpenSource03** [reply]
Thanks, I've updated the status.
